Bixal/move.mil
# convert the automatic hook script to a manual script

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes

This pull request…

- moves the automatically hooked script to be a manually invoked one

## Testing

To verify the changes proposed in this pull request…

1. Deploy a new version or start a new instance 
2. Check instance deploy logs that no post deploy script was triggered
